terraform/aws-accounts/cloud-platform-aws/vpc/eks/components/variables.tf

```diff
@@ -134,6 +134,16 @@ variable "pingdom_api_token" {
   description = "Pingdom API token"
 }
 
+variable "slack_bot_token" {
```

Comment:
@jaskaransarkaria it would be good to add info on how and where these tokens and webhooks are generated